src/main/java/seedu/address/logic/messages/FitDeleteCommandMessages.java

```diff
@@ -10,27 +10,28 @@ public class FitDeleteCommandMessages extends Messages {
 
     public static final String COMMAND_WORD = "fitdelete";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes exercise(s) from the client identified "
-        + "by their corresponding index.\n"
-        + "Parameters: INDEX (must be a positive integer) "
-        + PREFIX_EXERCISE_NAME + "EXERCISE_NAME\n"
-        + "Example: " + COMMAND_WORD + " 1 "
-        + PREFIX_EXERCISE_NAME + "squats\n"
-        + "Parameters: INDEX (must be a positive integer) "
-        + PREFIX_FITDELETE_DELETE_ALL + "\n"
-        + "Example: " + COMMAND_WORD + " 1 "
-        + PREFIX_FITDELETE_DELETE_ALL;
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the specified exercise from the client identified "
```

Comment:
"Deletes the exercise specified by name, from the client...." should be clearer.